packages/web-forms/src/demo/demo.ts

```diff
@@ -4,6 +4,9 @@ import { createApp } from 'vue';
 import { webFormsPlugin } from '../WebFormsPlugin';
 import OdkWebFormDemo from './OdkWebFormDemo.vue';
 
+import hankenGrotesk400 from '@fontsource/hanken-grotesk/400.css?inline';
+import hankenGrotesk600 from '@fontsource/hanken-grotesk/600.css?inline';
+import hankenGrotesk700 from '@fontsource/hanken-grotesk/700.css?inline';
```

Comment:
Should these also be configured in `vite.config.ts`? For Roboto, the config comment says:

> Load only a subset of the font family.

I can't remember if that's copypasta from something I found online, or if I made that note myself based on the effect of the config. If it does make a difference to build weight, I think it'd be worth it.

Reply:
I have made the change in https://github.com/getodk/web-forms/pull/223/commits/b3ae435863f5c09ea8068dcdc26396abd59e3bb7

https://www.npmjs.com/package/unplugin-fonts#import-generated-font-rules-css says to import `unfonts.css` if using custom or fontsource providers which we are.

So I have removed the manual import of fonts from `demo.ts` and `index.ts`; and imported `unfonts.css` in both. I hope this is the the right way. There is no impact on the bundle size of this change. I have also verified that fonts are loaded at runtime by running Demo app and @getodk/web-forms lib (`dist`) in Central.

Reply:
As mentioned in the Slack thread, this change unfortunately broke font loading in the build artifact. Feel free to cherry pick [this commit](https://github.com/getodk/web-forms/commit/56f64ee59db553beae3e3a37691801ae066275b7) reverting the change and removing `unplugin-fonts` entirely. Sorry for the diversion on this one!